dumpling-overview.md

```diff
@@ -318,9 +318,9 @@ When Dumpling is exporting a large single table from TiDB, Out of Memory (OOM) m
 + Reduce the value of `--tidb-mem-quota-query` to `8589934592` (8 GB) or lower. `--tidb-mem-quota-query` controls the memory usage of a single query statement in TiDB.
 + Adjust the `--params "tidb_distsql_scan_concurrency=5"` parameter. [`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) is a session variable which controls the concurrency of the scan operations in TiDB.
 
-### TiDB GC settings when exporting a large volume of data
+### TiDB GC settings when exporting a large volume of data (more than 1 TB)
```

Comment:
faq/manage-cluster-faq.md L430 is not aligned with [docs-cn/9907](https://github.com/pingcap/docs-cn/pull/9907/files#diff-99d1372119ae4ed697e8d7fda222067f376d7bc2881ddce7d4bec353c2d00b05).

Reply:
Done in https://github.com/pingcap/docs/pull/9206/commits/329cc3645bf3032c56b0291ae68f2946ba379edb